src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2344,6 +2354,9 @@ void Eltwise::initSupportedPrimitiveDescriptors() {
         inputPrecisions[i] = filterPrecision(inputPrecisions[i]);
     }
     outputPrecision = filterPrecision(outputPrecision);
+#if defined(OV_CPU_WITH_SHL)
+    }
+#endif
 #if defined(OV_CPU_WITH_ACL)
     }
 #endif
```

Comment:
Could you please explain why the previous lines leaded to problems with cpplint?
```
#if defined(OV_CPU_WITH_ACL) || defined(OV_CPU_WITH_SHL)
    }
#endif
```

Reply:
I think maybe there are some internal bugs inside `cpplint`. I have minimized the code to locate the problem:
```cpp
// Copyright 2024 BHbean

#define MACRO_X86 1
// #define MACRO_ARM 0

namespace ov {
int test() {
    int var;
#if defined(MACRO_X86)
    bool b = true;
    if (b) {
        var = 0;
    } else {
#endif
#if defined(MACRO_ARM)
    bool b = false;
    if (b) {
        var = 1;
    } else {
#endif
    var = 2;
#if defined(MACRO_X86) || defined(MACRO_ARM)
    }
#endif
    return 0;
}
}   // namespace ov

```

If `namespace ov {}` is removed, this piece of code is fine to `cpplint`:
```
$ cpplint test_cpplint.cpp
Done processing test_cpplint.cpp
```
However, if the `namespace ov {}` is added, the mismatch error will occur:
```
$ cpplint test_cpplint.cpp
test_cpplint.cpp:6:  Failed to find complete declaration of namespace ov  [build/namespaces] [5]
Done processing test_cpplint.cpp
Total errors found: 1
```